### ISSUE TITLE
yields TODO: explain this error

### ISSUE THREAD

**Issue by GaryB432** [svelte-add/svelte-add#228]
running `npm create @svelte-add/kit@latest app --with typescript+scss` yields `TODO: explain this error. debug info: output [ 'app2', 'typescript+scss' ] with undefined'] with undefined`

Windows 11

**Comment by SMEETT** [comment]
Same problem here.

**Comment by manuel3108** [comment]
For anyone in the future encountering this in the future, you are required to execute your command like this
```
npm create @svelte-add/kit@latest app -- --with typescript+scss
```
instead of this 
```
npm create @svelte-add/kit@latest app --with typescript+scss
```
Please not the two added dashes (--) in the first command, before passing any additional command line arguments.

Sadly this is a limitation of NPM, see here in example: [[Sending command line arguments to npm script](https://stackoverflow.com/questions/11580961/sending-command-line-arguments-to-npm-script)](https://stackoverflow.com/questions/11580961/sending-command-line-arguments-to-npm-script)

I'm going to create a PR for this in the near future.

**Comment by GaryB432** [comment]
using 

```
npm create @svelte-add/kit@latest app -- --with typescript+css
```

does not give the error from the original issue.  However it creates both a `vite.config.js` and a `vite.cofig.ts` (both a javascript and typescript version)

**Comment by manuel3108** [comment]
I have pushed a commit and released it with a more meaningful error message. 

Also, the other error you mention got fixed at the same time, see #234
